lib/transformations/utils.test.js

```diff
@@ -157,4 +157,11 @@ var a = { plugs: [] }
 			.filter(p => utils.findObjWithOneOfKeys(p, ['a'])).size()).toEqual(1);
 		});
 	});
+
+	describe('getRequire', () => {
+		it('should create a require statement', () => {
+			const require = utils.getRequire(j, 'filesys', 'fs');
+			expect(require).toMatchSnapshot();
```

Comment:
Fragments dont actually have `.toSource()` on them. :( Will try to figure out if this is possible some other way. I think I should be able to convert it to  a node somehow.